content/_includes/embed.macros.njk

```diff
@@ -380,18 +380,21 @@ params:
 {%- if src -%}
 {%- set config = 'img' | fromTaxonomy -%}
 {%- if not ('://' in src) -%}
+  {%- set fallback = config.sizes[sizes].fallback if config.sizes[sizes] else none -%}
   {%- set src = ('/assets/images/' + src) -%}
   {%- set imgSrcSet = srcset(src, width) -%}
   {%- set imgSrcSet = imgSrcSet if (imgSrcSet != "") else none -%}
-  {%- set respSrc = src | imgSuffix(config.fallback) -%}
+  {%- set respSrc = src | imgSuffix(fallback or config.fallback) -%}
+  {%- set configSizes = config.sizes[sizes].html if config.sizes[sizes] else none -%}
 {%- endif -%}
+{% set resize = width | imgSize(height, sizes) %}
 {%- set attrs = attrs | merge({
   'src': respSrc or src,
   'alt': alt,
   'srcset': imgSrcSet,
-  'width': width,
-  'height': height,
-  'sizes': (config.sizes[sizes] or sizes) if imgSrcSet else none
+  'width': resize.width,
+  'height': resize.height,
+  'sizes': configSizes or sizes
```

Comment:
`sizes` is a string here -- is it (e.g. `img sizes="foobar"`) a meaningful fallback, or should we just use use `configSizes` or `none`?

Reply:
While `sizes="foobar"` isn't useful, this is to allow either arbitrary or predefined sizes: 

- `configSizes` will grab e.g. `taxonomy.img.sizes['media'].html` (which is also a string) if `sizes == 'media'`
- Fallback to `sizes` means we can explicitly pass in any one-off sizes string like `'50vw'` or `'(min-width: 30em) 50vw, 100vw'`…